### DISCUSSION TITLE
Images don't showup on my readme files.

### DISCUSSION THREAD

**Discussion by justdeen** [community/community#142683] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

After uploading an image in my readme file it doesn't seem to show up after I've committed changes. Does anyone experience this also?

**Comment by thedynomo** [comment]
Might be your network or service provider. I always faced this issue but after changing my service provider the problem stopped completely.

**Comment by ebndev** [comment]
Hi @justdeen & @thedynomo, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. 

Thanks for understanding and we hope to see you participate in other discussions in the future!
